SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/DeploymentLauncher/DeploymentLauncher.cs

```diff
@@ -22,6 +23,9 @@ internal class DeploymentLauncher
 
         private const string CoordinatorWorkerName = "SimulatedPlayerCoordinator";
 
+        private const string CHINA_ENDPOINT_URL = "platform-cn-production.api.spatialoschina.com";
```

Comment:
I'm not 100% sure this is the right endpoint, didi @Helcaraxan confirm this was the one?

Reply:
This should have been `platform.api.spatialoschina.com`. The production environment does not appears in the URL.